scripts/ci/run_ci.sh

```diff
@@ -86,9 +86,10 @@ done
 if [ -n "$MAIN_CI" ]; then
 
 	# West handling
+	GIT_DIR=$(basename $PWD)
         pushd ..
```

Comment:
Maybe spaces->tabs could be fixed for the `pushd` and `popd` lines too, to unbreak the indentation on GitHub.

Reply:
I thought I fixed that to be `tabs`.

Reply:
I fixed it just in a follow on commit in the PR.